src/pages/tutors/CampusCourseTutorPage.tsx

```
@@ -0,0 +1,436 @@
+    e.preventDefault();
+    CampusTutorServiceHandler.registerAsTutor({
+      tutorName: newTutorName,
+      avatarUrl: "https://images.unsplash.com/photo-1534528741775-53994a69daeb?w=150&auto=format&fit=crop&q=80",
+      courseCode: newCourseCode,
+      courseTitle: newCourseTitle,
+      department: newDepartment,
+      gradeAchieved: newGrade,
+      hourlyRate: newHourlyRate,
+      verifiedStudent: true,
+      bio: newBio,
+      subjects: newSubjects.split(",").map((s) => s.strip ? s.strip() : s.trim()),
```

ESLint

[error] Replace `s.strip·?·s.strip()·:·s.trim(` with `(s.strip·?·s.strip()·:·s.trim()`

(prettier/prettier)